app/uploaders/court_images_uploader.rb

```diff
@@ -2,19 +2,17 @@
 
 class CourtImagesUploader < CarrierWave::Uploader::Base
 
-  # Include RMagick or MiniMagick support:
   include CarrierWave::RMagick
-  # include CarrierWave::MiniMagick
 
-  # Include the Sprockets helpers for Rails 3.1+ asset pipeline compatibility:
-  # include Sprockets::Helpers::RailsHelper
-  # include Sprockets::Helpers::IsolatedHelper
-
-  # Choose what kind of storage to use for this uploader:
-  #storage :fog
+  if Rails.env.development? || Rails.env.test?
```

Comment:
since this seems to be CarrierWave configuration, how about to move it to the [CarrierWave initializer](https://github.com/ministryofjustice/courtfinder/blob/old_prod_working/config/initializers/carrierwave.rb)?

It could be something like 

```ruby
CarrierWave.configure do |config|
...

  if Rails.env.development? || Rails.env.test?
    config.storage = :file
  else
    config.storage = :fog
  end
end
```